apps/application-system/api/src/app/modules/application/files/utils/pdf.ts

```diff
@@ -14,12 +14,12 @@ export async function generateResidenceChangePdf(
   expiry: string,
 ): Promise<Buffer> {
   const doc = new PDFDocument({
-    size: Constants.PAGE_SIZE,
+    size: PdfConstants.PAGE_SIZE,
     margins: {
-      top: Constants.VERTICAL_MARGIN,
-      bottom: Constants.VERTICAL_MARGIN,
-      left: Constants.HORIZONTAL_MARGIN,
-      right: Constants.HORIZONTAL_MARGIN,
+      top: PdfConstants.VERTICAL_MARGIN,
+      bottom: PdfConstants.VERTICAL_MARGIN,
+      left: PdfConstants.HORIZONTAL_MARGIN,
+      right: PdfConstants.HORIZONTAL_MARGIN,
     },
   })
```

Comment:
I dream about having a PDF generation service, which supports multiple types of PDFs, accepts data through a POST body and renders the PDF either with react-pdf or plain react website that gets saved to PDF with the puppeteer.

Maybe one day :)